feddlib/core/Mesh/MeshPartitioner_def.hpp

```diff
@@ -247,11 +250,13 @@ void MeshPartitioner<SC,LO,GO,NO>::readAndPartitionMesh( int meshNumber ){
     bool buildEdges = pList_->get("Build Edge List", true);
     bool buildSurfaces = pList_->get("Build Surface List", true);
 
+	// Adding surface as subelement to elements
     if (buildSurfaces)
         this->setSurfacesToElements( meshNumber );
     else
         meshUnstr->deleteSurfaceElements();
 
+	// Serially distributed elements
```

Comment:
What are "serially distributed elements"?

Reply:
Before partitioning mesh entities are not distributed to each processor and each processor has 'serial' distribution. I shall rephrase